geoip2/records.py

```diff
@@ -844,6 +854,7 @@ def __init__(
         is_anonymous: bool = False,
         is_anonymous_proxy: bool = False,
         is_anonymous_vpn: bool = False,
+        is_anycast: bool = False,
```

Comment:
I missed this before, but in this one spot I think we need to put it at the end of the argument list (after mobile_network_code), since this would otherwise technically be a breaking change if it were called with positional parameters. Sorry about that.

Reply:
good point 👍